### PR TITLE
Newsletter 261 translate in french

### PR DESCRIPTION
**Pull request by Copinmalin**

### CODE REVIEW

File: _posts/fr/newsletters/2023-07-26-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+      script][topic multisignature] basés sur [MuSig2][topic musig], le protocole de diffusion devra être mis à jour. Un
+      [sujet][topic channel announcements] a également été discuté lors de la précédente réunion en personne des développeurs LN
+      (voir [Newsletter #204][news204 gossip]) pour savoir s'il convient de faire une mise à jour minimale du protocole (appelée
+      v1.5 gossip) qui ajoute simplement la prise en charge des sorties P2TR, ou une mise à jour plus générale du protocole
+      (appelée v2.0) qui permet plus largement l'utilisation d'une signature valide pour n'importe quelle UTXO de n'importe quel
+      type pour les annonces. Permettre l'utilisation de n'importe quelle sortie signifie que la sortie utilisée pour annoncer
+      le canal est moins susceptible qu'aujourd'hui d'être la sortie réellement utilisée pour exploiter le canal, rompant ainsi
+      le lien public entre les sorties et le financement du canal.
+
+Une autre considération discutée était de savoir si une UTXO d'une valeur de _n_ devrait être autorisée à annoncer un canal
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2023-07-26-newsletter.md:59]: @Copinmalin the markdown formatting for this bullet and the ones that follow is off. Can you correct to match the formatting for the above few bullets?

File: _posts/fr/newsletters/2023-07-26-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+*[Bitcoin Stack Exchange][bitcoin.se] est l'un des premiers endroits où les contributeurs d'Optech cherchent des réponses à leurs questions, ou lorsque nous avons quelques moments libres pour aider les utilisateurs curieux ou confus. Dans cette rubrique mensuelle, nous mettons en évidence certaines des questions et réponses les plus votées publiées depuis notre dernière mise à jour.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Comment puis-je calculer manuellement (sur papier) une clé publique Bitcoin à partir d'une clé privée ?]({{bse}}118933)
+  Andrew Poelstra donne un aperçu des techniques de vérification par calcul manuel, comme [codex32][news239 codex32], avant de décrire comment une clé publique pourrait être dérivée à la main à partir d'une clé privée, un processus qu'il estime prendre au moins 1500 heures, même avec des optimisations du processus. {% assign timestamp="57:18" %}
+
+  - [Pourquoi y a-t-il 17 versions natives segwit ?]({{bse}}118974)
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2023-07-26-newsletter.md:102]:
```suggestion
- [Pourquoi y a-t-il 17 versions natives segwit ?]({{bse}}118974)
```

**Review by bitschmidty — CHANGES_REQUESTED** [review]

**Review by bitschmidty — COMMENTED** [review]
@Copinmalin the 

PTLCs et surpaiement redondant
Propositions d'atténuation des attaques de saturation des canaux
Engagements simplifiés
Le processus de spécification

bullets are still mis-formatted, can you correct?

**Review by murchandamus — COMMENTED** [review]
Looks like everything is indented correctly now

**Comment by murchandamus** [comment]
I checked the links, looked over it visually. The issue raised by @bitschmidty seems to be resolved. 
Altogether, I think this is good to go, so I’ll merge.

**Pull request merged**